src/rabbit.erl

```diff
@@ -240,6 +240,7 @@
 -define(BOOT_START_TIMEOUT,     300).
 %% 12 hours
 -define(BOOT_FINISH_TIMEOUT,    43200).
+%% 100ms
```

Comment:
If that is true, and `Iterations = div(BOOT_START_TIMEOUT, BOOT_STATUS_SAMPLING_INTERVAL)` in `do_wait_for_boot_to_Start/2`, then I read `BOOT_START_TIMEOUT` & `BOOT_START_TIMEOUT` to be in milliseconds, not seconds.

Reply:
The names / comments could be clearer, for sure.

Reply:
Corrected.